fades/logger.py

```diff
@@ -50,15 +51,20 @@ def set_up(verbose, quiet):
     handler.setFormatter(formatter)
 
     # and to the syslog
-    try:
-        handler = logging.handlers.SysLogHandler(address='/dev/log')
-    except:
-        # silently ignore that the user doesn't have a syslog active; can
-        # see all the info with "-v" anyway
-        pass
-    else:
-        logger.addHandler(handler)
-        formatter = logging.Formatter(FMT_SYSLOG)
-        handler.setFormatter(formatter)
+    for syslog_path in ('/dev/log', '/var/run/syslog'):
+        if not os.path.exists(syslog_path):
+            continue
+        if os.path.exists(syslog_path):
```

Comment:
you should remove this `if` (as it's validated just above) and dedent the rest of the try/except...

Thanks!

Reply:
@facundobatista whoops - a copy-paste error. Fixed and pushed again. Sorry!